vispy/io/datasets.py

```diff
@@ -33,3 +33,20 @@ def load_crate():
         256x256x3 crate image.
     """
     return np.load(load_data_file('orig/crate.npz'))['crate']
+
+
+def load_spatial_filters():
+    """Load spatial-filters kernel
+
+    Returns
+    -------
+    kernel : array
+        16x1024 16 interpolation kernel with length 1024 each.
+    names : list with respective interpolation names
```

Comment:
numpy docstring format
```
names : list of str
    Respective interpolation names, plus "Nearest" which does not require
    a filter but can still be used.
```
Then you can have `"Nearest"` at the end of the list, no fiddling required later.